reana-ui/src/pages/privacyNotice/PrivacyNotice.js

```diff
@@ -0,0 +1,326 @@
+            <Table.Cell>
+              REANA administrators and CERN Services administrators (Database on
+              demand service)
+            </Table.Cell>
+            <Table.Cell>User support and service operations</Table.Cell>
+          </Table.Row>
+          <Table.Row>
+            <Table.Cell>E-group membership</Table.Cell>
+            <Table.Cell>
+              REANA administrators and CERN Services administrators (Database on
+              demand service)
+            </Table.Cell>
+            <Table.Cell>User support and service operations</Table.Cell>
+          </Table.Row>
+          <Table.Row>
+            <Table.Cell>Account name</Table.Cell>
+            <Table.Cell>
+              REANA administrators and CERN Services administrators (Database on
+              demand service)
+            </Table.Cell>
+            <Table.Cell>User support and service operations</Table.Cell>
+          </Table.Row>
+          <Table.Row>
+            <Table.Cell>Access token</Table.Cell>
+            <Table.Cell>
+              REANA administrators and CERN Services administrators (Database on
+              demand service)
+            </Table.Cell>
+            <Table.Cell>User support and service operations</Table.Cell>
+          </Table.Row>
+          <Table.Row>
+            <Table.Cell>List of REANA jobs you created</Table.Cell>
+            <Table.Cell>
+              REANA administrators and CERN Services administrators (Database on
+              demand service)
+            </Table.Cell>
+            <Table.Cell>User support and service operations</Table.Cell>
+          </Table.Row>
+        </Table.Body>
+      </Table>
+
+      <p>
+        For more detailed information about personal data and privacy please
+        refer to the{" "}
+        <a
+          href="https://privacy.web.cern.ch/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Data Privacy web site
+        </a>
+        .
+      </p>
+      <p>
+        For questions regarding this Privacy Notice, please contact{" "}
+        <a href="mailto:reana-support@cern.ch">reana-support@cern.ch</a>.
+      </p>
+      <p>
+        For questions regarding personal data and privacy please contact the{" "}
+        <a
+          href="https://privacy.web.cern.ch/office-data-privacy-odp"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Office of Data Privacy
+        </a>
+        .
+      </p>
+      <p>
+        To request to exercise data subject rights please fill and submit the
+        following{" "}
+        <a
+          href="https://cern.service-now.com/service-portal?id=sc_cat_item&name=data-subject-rights&se=Data-Privacy"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          online form
+        </a>
+        .
+      </p>
+      <p>This Privacy Notice is subject to revision.</p>
+      <p>
+        <em>Last revision: 09-10-2020 18:04:55</em>
```

Comment:
One more thing:  Please alter to `2020-10-09 18:04:55` as we usually do YYYY-MM-DD.  Hoping that 10 is the month :wink: (We could even write 09-Oct-2020.)